### ISSUE TITLE
[BUG] onDelete and onUpdate are not generated in migration file

### ISSUE THREAD

**Issue by t0nka** [TiagoSilvaPereira/vemto-issues#92]
**Describe the bug**
in the AddForeignsToXXX migration files the onDelete() and onUpdate() instructions are not generated

**To Reproduce**
1. generate some 1-to-many model relations
2. fill the "ondelete" inputfield of the model foreign key with "CASCADE"
3. generate the code
4. run php artisan migrate

**generated migration**
the following lines are generated wrong
```
        Schema::table('tags', function (Blueprint $table) {
            $table
                ->foreign('post_id')
                ->references('id')
                ->on('posts');
        });
```

**Expected behavior**
the expected migration files should contain the following
```
        Schema::table('tags', function (Blueprint $table) {
            $table
                ->foreign('post_id')
                ->references('id')
                ->on('posts')
                **->onDelete('cascade');**
        });
```

this behavior appears in multiple projects, even in newly created ones.

**onUpdate()** is also not generated in corresponding migration files.

**Desktop**
 - OS: ubuntu 20.x
 - Version: vemto 0.7.8
 - mysql 8+

**Comment by TiagoSilvaPereira** [comment]
Hi @t0nka thank you for reporting... I'll fix it for the next version

**Comment by TiagoSilvaPereira** [comment]
Fixed on version 0.8.1
